mgmt/rest/server.go

```diff
@@ -299,8 +309,20 @@ func (s *Server) Name() string {
 	return "REST"
 }
 
-func (s *Server) SetAddress(addrString string) {
-	s.addrString = addrString
+func (s *Server) SetAddress(addrString string, dfltPort int) error {
+	restLogger.Info(fmt.Sprintf("Setting address to: [%v] Default port: %v", addrString, dfltPort))
+	// In the future, we could extend this to support multiple comma separated IP[:port] values
+	if strings.Index(addrString, ",") != -1 {
+		return serror.ErrBadAddress
```

Comment:
The error here isn't used anywhere.  Remove it entirely since the func you added in snapd `checkCmdLineFlags` already checks for this?